.rubocop.yml

```diff
@@ -4,6 +4,6 @@ Metrics/BlockLength:
 Documentation:
   Enabled: false
 LineLength:
-  Max: 99
+  Max: 150
```

Comment:
99 is the maximum line length, you must left as it.

Reply:
Done